corehq/apps/app_manager/views/download.py

```diff
@@ -54,13 +56,15 @@ def download_odk_media_profile(request, domain, app_id):
     if not request.app.copy_of:
         username = request.GET.get('username', 'unknown user')
         make_async_build.delay(request.app, username)
+    else:
+        request._always_allow_browser_caching = True
     return HttpResponse(
         request.app.create_profile(is_odk=True, with_media=True),
         content_type="commcare/profile"
     )
 
 
-@safe_download
+@safe_cached_download
```

Comment:
Thanks. I updated https://github.com/dimagi/commcare-hq/pull/17171/commits/23c9d85dc463fc3d7770e7c7d04f6757090ed4f6 to check that it is a build and that it's released. That way in the future we could even deploy this to prod and we won't have a cache full of test releases.